.github/workflows/ngsa-pr.yaml

```diff
@@ -0,0 +1,53 @@
+name: Build NGSA-Java-PR
+
+on:
+  pull_request:
+    types: [opened, reopened, synchronize, closed]
+
+    paths:
+      - 'Dockerfile'
+      - 'ngsa/src/main/java/**'
```

Comment:
This "**" is definitely the right usage of the wildcard, we want the expression to match on `/` as well, since there are nested java files in the repo.